website/docs/d/org_vdc.html.markdown

```diff
@@ -0,0 +1,70 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_org_vdc"
+sidebar_current: "docs-vcd-data-source-org-vdc"
+description: |-
+  Provides a organization VDC data source.
+---
+
+# vcd\_org\_vcd
+
+Provides a vCloud Director Organization VDC data source. A Organization VDC can be used to reference a VCD and use its 
+data within other resources or data sources.
+
+Supported in provider *v2.5+*
+
+## Example Usage
+
+```hcl
+data "vcd_org_vdc" "my-org-vdc" {
+  org     = "my-org"
+  name    = "my-vdc"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org` - (Optional, but required if not set at provider level) Org name 
+* `name` - (Required) Organization VDC name
+
+## Attribute Reference
```

Comment:
This section is somewhat misleading, because it states that some parameters are required, while some are optional. At the end, it just duplicates what resource has. Hence, what about not listing them at all, like we do in other data source docs? I mean just giving the link/reference:
> All the attributes defined in ...

If we decide to do this, we should do this in all data sources. @Didainius and @dataclouder your thoughts?

Reply:
Both ways could work for me although I am more of a fan for referencing to resource page. Our "read" functions force the coupling between resource and data source parameters and if we have attributes here and in resource it will make us maintain documentation in two places, which is easy to diverge if we need to change/update something.

Reply:
Well, in this case I updated doc to be inline with others. Also there little difference between normal resource - `delete_recursive` and `delete_force` aren't part of data source as they are user input.